tests/setup/test_interactive_env_setup.py

```
@@ -0,0 +1,511 @@
+    """Test prompt_input function with mocked input"""
+
+    @patch("builtins.input", return_value="test_value")
+    def test_prompt_input_basic(self, mock_input):
+        """Test basic input prompt"""
+        result = prompt_input("Enter value")
+
+        assert result == "test_value"
+        mock_input.assert_called_once()
+
+    @patch("builtins.input", return_value="")
+    def test_prompt_input_default(self, mock_input):
```

Ruff (0.14.4)

Unused method argument: `mock_input`

(ARG002)